dataquality/utils/setfit.py

```diff
@@ -62,6 +63,8 @@ def log_preds_setfit(
 
     logger_config = dq.get_data_logger().logger_config
     labels = logger_config.labels
+    epoch = 0 if epoch is None else epoch
```

Comment:
slightly cleaner. If epoch is None or 0, itll be set to 0, otherwise, itll be kept
```suggestion
    epoch = epoch or 0
```

Reply:
genius